database/gdb/gdb_model_builder.go

```diff
@@ -37,6 +37,9 @@ func (m *Model) Builder() *WhereBuilder {
 // getBuilder creates and returns a cloned WhereBuilder of current WhereBuilder if `safe` is true,
 // or else it returns the current WhereBuilder.
 func (b *WhereBuilder) getBuilder() *WhereBuilder {
+	if !b.model.safe {
+		return b
+	}
```

Comment:
I have added unit testing case, and all checks have passed

Reply:
@org-org-org I've check the logic of `getBuilder` and its usages, it might be forgot updating the comment of `getBuilder`. Is there any issue that would be caused by current behavior of `getBuilder`?

Reply:
Actually, it's not a big problem. It's just that I mistakenly thought that the builder was not chain-safe by default like the model, so I didn't use an equal sign to receive the where condition when adding it in the if statement for the builder. This caused the SQL statement to have unexpected results. It wasn't until I looked at the source code that I realized it was chain-safe by default, so I submitted a pull request because the documentation and actual behavior seemed to be different.
其实不是什么大问题，只是我误以为builder和model一样默认是非链式安全的，所以我在if语句里给builder加where条件时没有用等号接收，导致了sql语句是非预期的结果。我查看了源码后才发现默认是链式安全的，因为注释和实际好像不一样就提了个PR

Reply:
@org-org-org It might then update the comment, not the codes?